src/Apps/Artist/Components/ArtistMeta.tsx

```diff
@@ -5,6 +5,8 @@ import React, { Component } from "react"
 import { Link, Meta, Title } from "react-head"
 import { createFragmentContainer, graphql } from "react-relay"
 import { data as sd } from "sharify"
+import artistBio from "../SEOArtistTest/ArtistBioInsightTestGroup.js"
```

Comment:
Is this something that could be moved to MP once it becomes more permanent?

Reply:
Good points! So I just confirmed with Nicolette that they are only running this test for about a week! I think the suggestion to move the data to MP makes perfect sense after we get the results back from the test to see which title performs best for SEO. I'll add a ticket to track moving this work there (and deprecating this PR) to the Grow backlog. 

In the meantime, does anyone object to merging this PR now/no concerns about the code itself? This ticket was created months ago and I'd love to let the SEO team know that this work is ready for them to begin!

Reply:
I would still keep it in Metaphysics, even it's just for a week. Otherwise there's a large static file that needs to get bundled with Reaction, published to npm, and included in Force's `node_modules/`. Metaphysics is a great place for this sort of thing.

Up to you! I have no objections for merging this as-is.